platform-ui-skin/src/main/webapp/skin/less/social/skin/VersionHistoryDrawer/style.less

```diff
@@ -0,0 +1,99 @@
+/*
+This file is part of the Meeds project (https://meeds.io/).
+Copyright (C) 2022 Meeds Association
+contact@meeds.io
+This program is free software; you can redistribute it and/or
+modify it under the terms of the GNU Lesser General Public
+License as published by the Free Software Foundation; either
+version 3 of the License, or (at your option) any later version.
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+Lesser General Public License for more details.
+You should have received a copy of the GNU Lesser General Public License
+along with this program; if not, write to the Free Software Foundation,
+Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+*/
+@import "../../../variables.less";
+
+.VuetifyApp .v-application {
+  .versionHistoryDrawer {
+    .drawerContent {
+      .current_version {
+        background-color: @lightGrey;
```

Comment:
in general case, we use [helpers.less](https://github.com/Meeds-io/platform-ui/blob/develop/platform-ui-skin/src/main/webapp/skin/less/helpers.less#L40) (where you can define new generic classes if needed as well) &  [Vuetify classes](https://github.com/Meeds-io/platform-ui/blob/develop/platform-ui-skin/src/main/webapp/skin/less/vuetify/lib/vuetify.less#L21) instead of increasing CSS directives by defining a new one to apply a color, margin, text style, font style, display directive... In general case, we add in addons and specific portlets styles only fixed widths, heights ... workarounds about specific use cases in our portlet that aren't about a generic UX that should be applied globally on product.